### PR TITLE
Add Ship It Plugin

### PR DESCRIPTION
**Pull request by nemani**

### CODE REVIEW

File: plugins/ShipIt/ShipIt.plug [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[Core]
+Name = Ship it
+Module = ShipIt
+
+[Documentation]
+Description = A script to display a squireel.
```
Comment (meetmangukiya) [inline plugins/ShipIt/ShipIt.plug]: typo

File: plugins/ShipIt/ShipIt.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    Displays the Ship It squirel!
+    """
+
+    squirrels = [
+      r"http://shipitsquirrel.github.io/images/ship%20it%20squirrel.png",
+      r"http://images.cheezburger.com/completestore/2011/11/2/aa83c0c4-2123-4bd3-8097-966c9461b30c.jpg",
+      r"http://images.cheezburger.com/completestore/2011/11/2/46e81db3-bead-4e2e-a157-8edd0339192f.jpg",
+      r"http://28.media.tumblr.com/tumblr_lybw63nzPp1r5bvcto1_500.jpg",
+      r"http://i.imgur.com/DPVM1.png",
+      r"http://d2f8dzk2mhcqts.cloudfront.net/0772_PEW_Roundup/09_Squirrel.jpg",
```
Comment (meetmangukiya) [inline plugins/ShipIt/ShipIt.py]: why raw strings ?

**Review by meetmangukiya — DISMISSED** [review]

**Comment by meetmangukiya** [comment]
Also, can you test it in the test channel and link to that message ?

**Comment by meetmangukiya** [comment]
We are getting this project running again? :)

File: plugins/ShipIt/ShipIt.py (hunk 2 above)
Comment (nemani) [reply plugins/ShipIt/ShipIt.py]: To prevent the escape of `/`

Comment (nemani) [reply plugins/ShipIt/ShipIt.py]: I mean escape of all special characters

**Comment by nemani** [comment]
You want the link in the PR or as a comment in the code? 
Yes! I think this might be useful for `Real Time Communication Bridge` gsoc project.

File: plugins/ShipIt/ShipIt.py (hunk 2 above)
Comment (nemani) [reply plugins/ShipIt/ShipIt.py]: Actually unneeded.

**Comment by nemani** [comment]
Test message https://gitter.im/coala/errbot?at=58a732870ad50ac3152c4052

**Pull request merged**